### DISCUSSION TITLE
Editing Vault on multiple machines

### DISCUSSION THREAD

**Discussion by andytinkham** [oleeskild/obsidian-digital-garden#266] [Q&A] [answered]
I've got my digital garden set up on one laptop. There's an Obsidian vault, it pushes to a github repo, and publishes to vercel. Now, I'd like to be able to add the vault to a second laptop. I want to make sure my assumptions are correct.

1. Any mode of syncing the Obsidian vault across computers should work - Obsidian sync, Obsidian git. If I do it through git, that should be a separate repo than the one Vercel is publishing from. (I'll likely use Sync, since I already pay for it for my main vault).
2. Once the repo is synced, obsidian-digital-garden will continue to do its git stuff in the background. I should be able to publish pages from either machine and the sync process / digital-garden internal git logic is sufficient that I shouldn't be causing merge conflicts or losing data.

Am I thinking correctly about how this should work? Thank you!

**Comment by shinanqu** [comment, marked as answer]
I cannot answer directly to your situation because I neither use Sync nor Git. I am syncing with Dropbox. I edit my files on my mobile phone, as well as on my computer. If the sync is set up correctly, the version on the computer should be the same as the version on the mobile phone. There is a delay on the mobile phone because the sync doesn't happen instantly, but with additional apps (FolderSync in my case).

Then what is on your Digital Garden is another matter. You can add the front matter flag (dg-publish: true) and trigger the publish command. With a shortcut, both will happen at the same time. You write something, you hit the shortcut, and it's online on your homepage.

You can do it on your computer, you can do it on your mobile phone. Only the files get published, which have the flag and where the publish command was triggered (though you can also trigger it for all notes containing the flag at the same time).

When you edit files, nothing will happen with your homepage, nothing will be synced. Only if you trigger the publish command again, your homepage will be updated. So, it's all about the publish command, which can be triggered on your mobile phone, as well as on your computer or second laptop.

What I want to say, and hopefully, it answers your questions somehow... The publishing on the digital garden happens manually. It is independent of your vault sync setup. No matter if you have Sync or Git or if it's working or if you have multiple versions of the vault, which are not correctly synced etc. It all doesn't matter for the Digital Garden. The Digital Garden only cares about the notes, which you manually publish, no matter from which device.

**Reply by David7ce** [reply]
Obsidian Digital Garden is a plugin that use 11ty (Static Site Generator) with a theme and some extra things.

The plugin does not generate the output in local it connects to GitHub and Vercel and deploy the generated website via API.

So the only things that matter to sync a vault is copy the vault including the plugin. You can use any software like rsync, git, freefilesync, synching, etc.

One thing that I miss is that Obsidian DG generate the 11ty static website locally. And then deploy the website manually.

**Reply by andytinkham** [reply]
Thank you both! I missed the original answer back in April, but I do have things working as I expected now, across multiple machines!
